### PR TITLE
Fixed dashboard icons to match zeplin and added spacing to dashboard …

### PR DESCRIPTION
**Pull request by kevinylee1991**
…spinner
Screen Shot 2019-12-16 at 5 49 03 PM

### CODE REVIEW

File: src/app/plugins/kubernetes/components/dashboard/StatusCard.tsx [hunk 1]
```diff
@@ -112,9 +113,11 @@ const StatusCard: FunctionComponent<StatusCardProps> = ({ entity, route, addRout
           </Link>
         </div>
         <div className={clsx(rowColumn, verticalCenter)}>
-          {iconComponent}
+          {iconComponent(icon)}
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/dashboard/StatusCard.tsx]: Dont need a function here, just render the font awesome icon `<FontAwesomeIcon className={headerIcon}>{icon}</FontAwesomeIcon>`

File: src/app/plugins/kubernetes/components/dashboard/StatusCard.tsx [hunk 2]
```diff
@@ -112,9 +113,11 @@ const StatusCard: FunctionComponent<StatusCardProps> = ({ entity, route, addRout
...
           {loading ? (
-            <CircularProgress size={32} />
+            <div>
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/dashboard/StatusCard.tsx]: is this div just an extra element or is it doing something specific?

File: src/app/plugins/kubernetes/components/dashboard/StatusCard.tsx [hunk 3]
```diff
@@ -13,10 +13,12 @@ import FontAwesomeIcon from 'core/components/FontAwesomeIcon'
 
 const useStyles = makeStyles((theme: any) => ({
   headerIcon: {
-    width: theme.spacing(5),
-    height: theme.spacing(5),
+    fontSize: '36px',
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/dashboard/StatusCard.tsx]: I'm ok with this font-size if its not one of the variants `font-awesome` supplies. Does it not match `2x` or `3x` sizing?

Comment (kevinylee1991) [reply src/app/plugins/kubernetes/components/dashboard/StatusCard.tsx]: I just checked and it's somewhere between 2x and 3x

Comment (jruddell) [reply src/app/plugins/kubernetes/components/dashboard/StatusCard.tsx]: ok sounds good!

**Pull request merged**